### PR TITLE
fix: revise  markdown errors

### LINKED ISSUE #106: docs: Found some Markdown errors
### Describe the bug

I found some Markdown errors in the "Welcome to Community" file. This something I will work on. 

### Steps to reproduce

N/A

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by CBID2**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [X] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR fixes the Markdown errors found in one of the docs. 
## Related Tickets & Documents

Fixes #106 
## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [X] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
Looks good to me

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -15,4 +15,24 @@ OpenSauced provides guidance for new contributors finding their next contributio
 
 The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions whether it's an article you wrote for [Codecademy's Docs website](https://www.codecademy.com/pages/contribute-docs), revising one of the lessons on [freeCodeCamp's curricula](https://contribute.freecodecamp.org/#/index?id=learning-platform), or creating a tutorial for people who use [Audacity](https://support.audacityteam.org/community/contributing/tutorials).
 
-Eager to get started? Check out [the Effectively Highlights section in our free "Intro to Open Source" course](https://github.com/open-sauced/intro/blob/main/06-the-secret-sauce.md#effectively-highlight-your-contributions) to learn more.
+Eager to get started? Check out[the Effectively Highlights section in our free "Intro to Open Source" course](https://github.com/open-sauced/intro/blob/main/06-the-secret-sauce.md#effectively-highlight-your-contributions) to learn more.
```
Comment (BekahHW) [inline docs/introduction.md:18]: I would add the full title here:

> Check out the "Effectively Highlight Your Contributions" section...

**Comment by CBID2** [comment]
Made the change @BekahHW! :)

File: docs/introduction.md [hunk 2]
```diff
@@ -9,10 +9,30 @@ sidebar_label: Introduction to OpenSauced 🍕
...
+## FAQs
+
+- Add information about FAQs here.
+
+## Community
+
+- Add information about the community here.
+  
+## Contributing  
+
```
Comment (bdougie) [inline docs/introduction.md:37]: These sections are not needed in this PR. Everything else looks great

**Review by bdougie — CHANGES_REQUESTED** [review]

File: docs/introduction.md (hunk 2 above)
Comment (bdougie) [inline docs/introduction.md:37]:
```suggestion
```

**Review by bdougie — APPROVED** [review]

**Pull request merged**